### PR TITLE
prevent workflows running on forks - fix

### PR DESCRIPTION
**Pull request by orenc1**
expressions in 'if' statements must be in paranthesis. tested on my private repo.

Signed-off-by: orenc1 <ocohen@redhat.com>

**Release note**:

```release-note
none
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: .github/workflows/rebase-bot.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 on:
   issue_comment:
     types: [created]
-name: Rebase Bot
 jobs:
   rebase:
-    name: Rebase
-    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase') && github.repository == ‘kubevirt/hyperconverged-cluster-operator’
+    name: Build Manifests and Rebase
+    if: (github.event.issue.pull_request != '') && (contains(github.event.comment.body, '/rebase')) && (github.repository == 'kubevirt/hyperconverged-cluster-operator')
```
Comment (nunnatsa) [inline .github/workflows/rebase-bot.yml:8]: Shouldn't the whole if statement be in parenthesis? e.g.
```
if: ((github.event.issue.pull_request != '') && (contains(github.event.comment.body, '/rebase')) && (github.repository == 'kubevirt/hyperconverged-cluster-operator'))
```

Comment (orenc1) [reply .github/workflows/rebase-bot.yml:8]: I checked it on my fork and it works like it's written.
https://github.com/orenc1/hyperconverged-cluster-operator/blob/master/.github/workflows/rebase-bot.yml#L8
https://github.com/orenc1/hyperconverged-cluster-operator/actions/runs/127581061

**Comment by orenc1** [comment]
test

**Pull request merged**